app/src/main/java/de/westnordost/streetcomplete/quests/tactile_paving/TactilePavingAnswer.kt

```diff
@@ -0,0 +1,7 @@
+package de.westnordost.streetcomplete.quests.tactile_paving
+
+enum class TactilePavingAnswer(val osmValue: String) {
```

Comment:
It would be consistent if this was named `TactilePavingCrosswalkAnswer`. Not a merge blocker though.

Reply:
Done